libraries/Microsoft.Bot.Builder.Core.Extensions/MemoryStorage.cs

```diff
@@ -12,15 +12,15 @@ namespace Microsoft.Bot.Builder.Core.Extensions
     /// <summary>
     /// Models IStorage around a dictionary 
     /// </summary>
-    public class DictionaryStorage : IStorage
+    public class Dictionary : IStorage
```

Comment:
Having a class have the same name as a Dictionary is probalby not a good idea. 

Please rename.

Reply:
this is a rename corruption - nice catch!